src/cursor/cur_hs.c

```diff
@@ -998,14 +997,17 @@ __curhs_insert(WT_CURSOR *cursor)
     } while ((ret = __wt_hs_modify(cbt, hs_upd)) == WT_RESTART);
     WT_ERR(ret);
 
-    /* We no longer own the update memory, the page does; don't free it under any circumstances. */
+    /*
+     * We no longer own the update memory, the page does; don't free it under any circumstances.
```

Comment:
Why did this get multi-lined?

Reply:
Not sure, s_all did that but I've reverted it: [4ec1860](https://github.com/wiredtiger/wiredtiger/pull/8623/commits/4ec1860657e6aeef19c7f7a20fa12817c7c791a1)